src/pages/tipz/index.tsx

```diff
@@ -6,7 +6,7 @@ import Balancer from 'react-wrap-balancer'
 import {Card, CardContent, CardHeader} from 'ui'
 
 export async function getStaticProps() {
-  const tips = await getAllTips()
+  const tips = await getAllTips({onlyPublished: true})
```

Comment:
updating this function call to only grab published tips which allows us to remove the filter below